src/pdfboxing/util.clj

```diff
@@ -0,0 +1,38 @@
+(ns pdfboxing.util)
+
+(defn first-line-valid?
```

Comment:
@lucaswiman possibly. One thing that I didn't add, but probably should, is warnings on each of the checks, so that it's obvious where the failure occurred. Because currently it's a bit of a black box. 

If I have it as a regular expression, then it'll still stay as a black box.

What do you think?

Thanks for the comment! :)